File: 1341-split-a-string-in-balanced-strings/split-a-string-in-balanced-strings.py
class Solution:
    def balancedStringSplit(self, s: str) -> int:
        count = 0
        res = 0
        for i in s :
            if i == 'R':
                count += 1
            else :
                count -= 1
            if count == 0:
                res += 1
        return res

File: 1341-split-a-string-in-balanced-strings/split-a-string-in-balanced-strings.dart
class Solution {
  int balancedStringSplit(String s) {
    int count = 0;
    int res = 0;
    for(String str in s.split('')){
      str == "R" ? count++ : count--;
      if(count == 0) res++;
    }
    return res;
  }
}